conf/parsers.conf

```diff
@@ -83,10 +83,10 @@
     Time_Key time
 
 [PARSER]
-    # http://rubular.com/r/izM6olvshn
-    Name crio
+    # http://rubular.com/r/tjUt3Awgg4
+    Name cri
     Format Regex
-    Regex /^(?<time>.+)\b(?<stream>stdout|stderr)\b(?<log>.*)$/
+    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$
     Time_Key    time
     Time_Format %Y-%m-%dT%H:%M:%S.%N%:z
```

Comment:
Is this okay? `%N` and `%:z`
`%Y-%m-%dT%H:%M:%S.%N%:z`

Should it be this?
`%Y-%m-%dT%H:%M:%S.%L%z`